pages/users/user_reactivate.php

```diff
@@ -0,0 +1,56 @@
+<?php
+restrict_access();
+$can_add_user = check_auth(Access::$EDIT_USERS);
+$users_repository = em()->getRepository(User::class);
+page("Réactiver des licenciés")->css("user_list.css");
+if (!empty($_POST)) {
+    foreach (array_keys($_POST) as $key) {
+        $user = $users_repository->findOneBy(['id' => $key]);
```

Comment:
This is safe from injection, but it is not performant. If you want you can inspire yourself from my PR! Otherwise I would say this is good enough, we can refactor this later if need be 🙂